src/test/java/seedu/address/model/step/UniqueStepListTest.java

```diff
@@ -0,0 +1,150 @@
+package seedu.address.model.step;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.step.exceptions.DuplicateStepException;
+import seedu.address.model.step.exceptions.StepNotFoundException;
+
+
+public class UniqueStepListTest {
+
+    private final UniqueStepList uniqueStepList = new UniqueStepList();
+    private static final Step TESTSTEP = new Step("Cook 1 egg."); // for the time being only
+    private static final Step TESTSTEP2 = new Step("Boil 2 egg."); // for the time being only
```

Comment:
Static variables should be at the top of the class. Can just shift it to be before ```private final UniqueStepList...``` I believe

Reply:
fixed